addons/cppcheckdata.py

```diff
@@ -689,7 +687,7 @@ def ArgumentParser():
     return parser
 
 
-def reportError(template, callstack=[], severity='', message='', id=''):
+def reportError(template, callstack=(), severity='', message='', id=''):
```

Comment:
If you change default arguments in-place, next time default argument is not "default".

Simple example:
```
def f(l=[]):
    print(l)
    l.append(2)

f()
f()
```

It prints:
```
[]
[2]
```

That's why it's not a very good practice to use mutable default arguments.